src/main/java/MessageUserCase/MessageResponseModel.java

```diff
@@ -0,0 +1,26 @@
+package MessageUserCase;
+
+public class MessageResponseModel {
+    String send;
+
+    String creationTime;
+
+    public MessageResponseModel(String send, String creationTime) {
+        this.send = send;
+        this.creationTime = creationTime;
+    }
+
+
```

Comment:
You need to use the method called"blockuser" in the report part when user receive messages since we should check if messages contain words in sensitive word list.